### PR TITLE
Update GenreMDB property access levels

### PR DESCRIPTION
**Pull request by MartinP7r**
Sorry if I'm wrong, but I think these props are not accessible without `public` declaration.

### CODE REVIEW

File: Sources/TMDBSwift/OldModels/ExternalIdsMDB.swift [hunk 1]
```diff
@@ -9,10 +9,10 @@
...
-    public var  id: Int!
-    public var  freebase_id: String?
-    public var  tvdb_id: Int?
-    public var  tvrage_id: Int?
+    public var imdb_id: String?
+    public var freebase_mid: String?
+    public var id: Int!
+    public var freebase_id: String?
+    public var tvdb_id: Int?
+    public var tvrage_id: Int?
```
Comment (MartinP7r) [inline Sources/TMDBSwift/OldModels/ExternalIdsMDB.swift:17]: ## Review memo

Unrelated formatting fix.

**Review by gkye — APPROVED** [review]
